plugins/modules/ec2_vpc_vgw.py

```diff
@@ -240,17 +265,13 @@ def attach_vgw(client, module, vpn_gateway_id):
     return result
 
 
-def detach_vgw(client, module, vpn_gateway_id, vpc_id=None):
-    params = dict()
-    params["VpcId"] = module.params.get("vpc_id")
+def detach_vgw(client, module: AnsibleAWSModule, vpn_gateway_id: str, vpc_id: Optional[str] = None) -> Any:
+    vpc_id = vpc_id or module.params.get("vpc_id")
```

Comment:
I check_mode handled for this operation?

Reply:
fixed